### ISSUE TITLE
Issue: Leave last item

### ISSUE THREAD

**Issue by Karmacopper** [MSchmoecker/ValheimHopper#15]
This appears to be retaining the last item in the hopper rather than the chest it pulled from.

Reading the change log led me to expect an item left in the chest. Which would be useful in conjunction with e.g. [AzuAutoStore](https://thunderstore.io/c/valheim/p/Azumatt/AzuAutoStore/)

**Comment by MSchmoecker** [comment]
Yeah that's a fair point. The idea behind the current behavior is that you often don't place a chest, if only a handful amount of materials are processed/moved.
However, I definitely see that this should also apply to chests. I think a good compromise would be to leave the last item in the hopper if no chest is on top, and use the chest otherwise.

**Comment by Karmacopper** [comment]
Agreed. :D
